mava/utils/jax_training_utils.py

```diff
@@ -46,42 +49,88 @@ def compute_running_mean_var_count(
     M2 = m_a + m_b + jnp.square(delta) * count * batch_count / tot_count
     new_var = M2 / tot_count
     new_count = tot_count
+    new_std = jnp.sqrt(new_var)
 
-    return jnp.array([new_mean, new_var, new_count])
+    return dict(mean=new_mean, var=new_var, std=new_std, count=new_count)
```

Comment:
Is there a reason you changed this to a dict? In general jax arrays will keep more of the code jittable, although sometimes less readable. Don't feel like you have to change this because it'll probably be a big change, but in general I think keeping jax arrays as much as possible is a good idea

Reply:
Because I wanted to use the same functions for observation normalisation and target value normalisation. The observations are multidimensional so we can just stack them in array like for the target values where we only have single value values for the mean, var and ...

Reply:
I think you could use a 3x1 array then? But again this isn't a big issue so I think you can leave it as is :smile:

Reply:
I will keep like this for now.